tests/goth_tests/test_mid_agreement_payments/test_mid_agreement_payments.py

```diff
@@ -1,43 +1,52 @@
 """A goth test scenario for mid-agreement payments."""
-from functools import partial
+from goth.assertions import EventStream
```

Comment:
hmm.... I'd say that imports from `goth` are our internal imports and should not be mingled with the third-party ones... ?